apps/native-component-list/src/navigation/MainNavigators.tsx

```diff
@@ -29,4 +33,13 @@ export default {
     },
     navigator: ExpoComponentsStackNavigator,
   },
-};
+} as Record<
+  string,
+  | undefined
+  | {
+      linking: any;
```

Comment:
Shouldn't it be a known type? I mean, isn't `react-navigation` exposing some useful typings?`

Reply:
Good call, it's `PathConfig`.